src/UnityExtension/Assets/Editor/GitHub.Unity/UI/ProjectWindowInterface.cs

```diff
@@ -11,18 +11,20 @@ class ProjectWindowInterface : AssetPostprocessor
         private const string AssetsMenuRequestLock = "Assets/Request Lock";
         private const string AssetsMenuReleaseLock = "Assets/Release Lock";
         private const string AssetsMenuReleaseLockForced = "Assets/Release Lock (forced)";
-        private static readonly List<GitStatusEntry> entries = new List<GitStatusEntry>();
+
+        private static List<GitStatusEntry> entries = new List<GitStatusEntry>();
         private static List<GitLock> locks = new List<GitLock>();
+        private static List<string> guids = new List<string>();
+        private static List<string> guidsLocks = new List<string>();
 
-        private static readonly List<string> guids = new List<string>();
-        private static readonly List<string> guidsLocks = new List<string>();
         private static IApplicationManager manager;
         private static bool isBusy = false;
         private static ILogging logger;
         private static ILogging Logger { get { return logger = logger ?? LogHelper.GetLogger<ProjectWindowInterface>(); } }
         private static CacheUpdateEvent lastRepositoryStatusChangedEvent;
         private static CacheUpdateEvent lastLocksChangedEvent;
-        private static IRepository Repository { get { return manager.Environment.Repository; } }
+        private static IRepository Repository { get { return manager != null ? manager.Environment.Repository : null; } }
+        private static bool IsInitialized { get { return Repository != null && Repository.CurrentRemote.HasValue; } }
```

Comment:
I understand why this is named `IsInitialized` because of how it is used in `EnsureInitialized`, but it could probably be better named as `HasRepositoryAndRemote`.